deepchem/models/tensorgraph/models/graph_conv.py

```diff
@@ -0,0 +1,163 @@
+import tensorflow as tf
+from deepchem.models.tensorgraph.tensor_graph import TensorGraph
+from deepchem.models.tensorgraph.layers import Input, Dense, Concat, SoftMax, SoftMaxCrossEntropy, Layer, \
+  GraphConvLayer, BatchNormLayer, GraphPoolLayer, GraphGather, WeightedError
+from deepchem.metrics import to_one_hot
+from deepchem.feat.mol_graphs import ConvMol
+import time
+
+
+class GraphConvTensorGraph(TensorGraph):
+  """
+  """
+
+  def __init__(self, **kwargs):
+    super().__init__(**kwargs)
+    self.min_degree = 0
+    self.max_degree = 10
+
+  def _construct_feed_dict(self, X_b, y_b, w_b, ids_b):
```

Comment:
Good point! I have a thought for a slight modification to the API that should catch this case. Will make an update to #498